python_modules/dagster/dagster/core/asset_defs/decorators.py

```diff
@@ -22,6 +23,7 @@ def asset(
     description: Optional[str] = None,
     required_resource_keys: Optional[Set[str]] = None,
     io_manager_key: Optional[str] = None,
+    compute_kind: Optional[str] = None,
```

Comment:
I think it would ultimately be useful to have both.  E.g. could later on support a "kind" or "asset_kind" field that expects values like "table", "ml_model", "dashboard", etc.